tasks/main.yml

```
@@ -35,20 +35,13 @@
     state: touch
     mode: 0644
 
-- name: Load tasks file with setup tasks based on the OS type
-  ansible.builtin.include_tasks: "{{ lookup('first_found', params) }}"
-  vars:
-    params:
-      files:
-        - setup_{{ ansible_distribution }}_{{ ansible_distribution_release }}.yml
-        # The ansible_distribution_release variable is always empty
-        # for Fedora, so this is necessary since we want to do
-        # something different for Fedora 34.
-        - setup_{{ ansible_distribution }}_{{ ansible_distribution_version }}.yml
-        - setup_{{ ansible_distribution }}.yml
-        - setup_{{ ansible_os_family }}.yml
-      paths:
-        - "{{ role_path }}/tasks"
+- name: Load tasks file for systemd setup
+  ansible.builtin.include_tasks: setup_systemd.yml
+  when: ansible_service_mgr == 'systemd'
+
+- name: Load tasks file for bin setup
+  ansible.builtin.include_tasks: setup_bin.yml
```

I think we should consider renaming `setup_bin.yml` to `setup_manual.yml`.  The latter more clearly indicates the purpose of the file (IMO).
```suggestion
- name: Load tasks file for manual setup
  ansible.builtin.include_tasks: setup_manual.yml
```

---

We also need to add some sort of documentation about this file and what it should contain.

---

I did the changes mentioned above in [e485997](https://github.com/cisagov/ansible-role-clamav/pull/39/commits/e4859970e9b3a4cbaa125b75501620b9e038f4ac).
Can you check it's all right ?

---

Looks good to me.  Thanks for making the changes.  @dav3r and @mcdonnnj, can you guys take another look?

---

I'm satisfied with the recent changes for this - thanks.

---

The updates work for me as well.